charts/auto-scaling-runner-set/templates/_helpers.tpl

```diff
@@ -208,18 +208,18 @@ env:
           {{- end }}
```

Comment:
hide whitespace when reviewing.

TLDR, the bug is `{{- end }}` get added in the wrong line.